CondFormats/L1TObjects/interface/L1TGlobalPrescalesVetos.h

```diff
@@ -18,7 +18,7 @@ class L1TGlobalPrescalesVetos {
   }
 
   unsigned int version_;
-  std::vector<std::vector<int> > prescale_table_;
+  std::vector<std::vector<double> > prescale_table_;
```

Comment:
@hjkwon260 You can't change data attributes types/names, since the back-ward compatibility in reading existing data would be broken. 
If this change is required, you should create a new class.

Reply:
@ggovi  thank you very much for the comment! I see, I'll create a new class and propagate it to o2o producer